packages/mobile/src/identity/contactMapping.ts

```diff
@@ -224,8 +221,8 @@ export function* fetchAddressesAndValidateSaga({
 function* getAccountAddresses(e164Number: string) {
   const phoneHashDetails: PhoneNumberHashDetails = yield call(fetchPhoneHashPrivate, e164Number)
   const phoneHash = phoneHashDetails.phoneHash
-  const lookupResult: IdentifierLookupResult = yield call(lookupAttestationIdentifiers, [phoneHash])
-  return getAddressesFromLookupResult(lookupResult, phoneHash) || []
+  const lookupResult: string[] = yield call(lookupAttestationIdentifiers, phoneHash)
+  return yield call(getAddressesFromLookupResult, lookupResult, phoneHash) || []
```

Comment:
Correct me if I am wrong but I don't think you need that `|| []` anymore because you've changed the output of `getAddressesFromLookupResult` to be an empty array instead of `null`